lib/iris/tests/test_merge.py

```diff
@@ -145,81 +145,143 @@ def _make_cube(self, data, dtype=np.dtype('int32'), fill_value=None,
         cube.add_aux_coord(height)
         return cube
 
+    @staticmethod
+    def _expected_fill(fill0, fill1):
+        result = None
+        if fill0 == fill1:
+            result = fill0
+        return result
+
+    def _check_fill_value(self, result, fill0, fill1):
+        fill_value = self._expected_fill(fill0, fill1)
```

Comment:
Rename to be `expected_fill_value` or `expected_cube_fill_value`?
May make it easier to follow